pywinauto/base_wrapper.py

```diff
@@ -377,7 +372,7 @@ def is_child(self, parent):
     #-----------------------------------------------------------
     def __eq__(self, other):
         "Returns true if 2 BaseWrapper's describe 1 actual element"
-        if hasattr(other, "_elementInfo"):
+        if hasattr(other, "_element_info"):
             return self.element_info == other.element_info
```

Comment:
Not sure if I understand it. ``` other``` is tested for  the ```_element_info``` attribute but compared against```other.element_info```